db/migrate/20170710145821_add_confirmable_to_user.rb

```diff
@@ -0,0 +1,13 @@
+class AddConfirmableToUser < ActiveRecord::Migration
+  def up
+    add_column :spree_users, :confirmation_token, :string
+    add_column :spree_users, :confirmed_at, :datetime
+    add_column :spree_users, :confirmation_sent_at, :datetime
```

Comment:
These `confirm_` look like a code smell. I'd rather create the `confirmations` table and them in there withour the prefix.